crates/biome_css_formatter/Cargo.toml

```diff
@@ -20,6 +20,7 @@ biome_rowan       = { workspace = true }
 biome_suppression = { workspace = true }
 
 [dev-dependencies]
+biome_configuration  = { workspace = true }
```

Comment:
Don't use the workspace for dev dependencies. Cargo will try to pull the version from crates.io

Reply:
Oh, thank you!
fixed